akka-remote/src/main/scala/akka/remote/artery/ObjectPool.scala

```diff
@@ -0,0 +1,24 @@
+/**
+ * Copyright (C) 2016 Lightbend Inc. <http://www.lightbend.com>
+ */
+package akka.remote.artery
+
+import org.agrona.concurrent.ManyToManyConcurrentArrayQueue
```

Comment:
I think we should add a direct dependency on Agrona as we are using it first hand, rather than just need it as a transitive dependency.